py_mmd_tools/mmd_elements.yaml

```diff
@@ -152,31 +147,32 @@ use_constraint:
   resource:
     maxOccurs: '1'
     acdd_ext: license_resource
+  license_text:
```

Comment:
Is this supposed to be empty?

Reply:
yes, I don't think we need it as long as we use one of the intended licenses. `license_text` is present to allow ingestion of datasets without a standard license, and at least the met datasets should have the CC-BY license.